src/lib/components/List/List.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+	import { type Variant, listVariants } from '../../../styles/tailwind/list';
+	import { typography } from '../../../styles/tailwind/typography';
+
+	export let title: string;
+	export let createdAt: string;
+	export let createdBy: string;
+	export let linkHref: string;
+	export let variant: Variant;
+
+	function formatDate(dateString: string): string {
+		const date = new Date(dateString);
+		const options: Intl.DateTimeFormatOptions = {
+			day: '2-digit',
+			month: 'long',
+			year: 'numeric',
+			calendar: 'buddhist'
+		};
+		return new Intl.DateTimeFormat('th-TH', options).format(date);
+	}
```

Comment:
you can use `day-js` to implement this for you instead of creating your own function